apps/hpc/hexagon/CradleCFD/run-scFlow.pbs

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+#PBS -l select=4:ncpus=120:mpiprocs=96:mem=420gb
+#PBS -l place=excl
+#PBS -N scFlow
+#PBS -j oe
+
+set -x
+
+ulimit -s unlimited
+ulimit -l unlimited
+ulimit -a
+
+sudo yum install -y python3
```

Comment:
Can we add a requirements section or a add_reqs.sh script that contains the packages that need to be installed instead of having them in the run scripts?